From: Andrew Grieve <agrieve@google.com>
Date: Wed, 28 Aug 2024 22:44:08 -0400
Subject: (cherry-pick-branch-heads/6668) Fix license metadata for
 spl_sqrt_floor, portaudio, sigslot

(cherry picked from commit 6ea1c96325baada0e6ba0b29456e02f403e15a1e)

Bug: b/361140175
Change-Id: I35e76039608fa5094c04ace5f3ad1dba868ccb85
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/360900
Reviewed-by: Henrik Andreassson <henrika@webrtc.org>
Commit-Queue: Andrew Grieve <agrieve@google.com>
Cr-Original-Commit-Position: refs/heads/main@{#42885}
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/361520
Reviewed-by: Mirko Bonadei <mbonadei@webrtc.org>
Reviewed-by: Harald Alvestrand <hta@webrtc.org>
Commit-Queue: Mirko Bonadei <mbonadei@webrtc.org>
Cr-Commit-Position: refs/branch-heads/6668@{#1}
Cr-Branched-From: 2cfedb277ae8dd2a8d8dd68aca6f95081c265671-refs/heads/main@{#42810}
---
 common_audio/third_party/spl_sqrt_floor/README.chromium | 2 +-
 modules/third_party/portaudio/README.chromium           | 2 +-
 rtc_base/third_party/sigslot/README.chromium            | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/common_audio/third_party/spl_sqrt_floor/README.chromium b/common_audio/third_party/spl_sqrt_floor/README.chromium
index b2c4309bd6..2ecaee5537 100644
--- a/common_audio/third_party/spl_sqrt_floor/README.chromium
+++ b/common_audio/third_party/spl_sqrt_floor/README.chromium
@@ -3,7 +3,7 @@ Short Name: sql_sqrt_floor
 URL: http://www.pertinentdetail.org/sqrt
 Version: 0
 Date: 2018-03-22
-License: Custom license
+License: SPL-SQRT-FLOOR
 License File: LICENSE
 Security Critical: yes
 Shipped: yes
diff --git a/modules/third_party/portaudio/README.chromium b/modules/third_party/portaudio/README.chromium
index 3f7beef655..d912d912e2 100644
--- a/modules/third_party/portaudio/README.chromium
+++ b/modules/third_party/portaudio/README.chromium
@@ -3,7 +3,7 @@ Short Name: portaudio
 URL: https://github.com/PortAudio/portaudio/tree/master/src/common
 Version: 9d8563100d841300f1689b186d131347ad43a0f6
 Date: 2022-04-12
-License: Custom license
+License: MIT
 License File: LICENSE
 Security Critical: yes
 Shipped: yes
diff --git a/rtc_base/third_party/sigslot/README.chromium b/rtc_base/third_party/sigslot/README.chromium
index 9867dd0320..98eac55f9a 100644
--- a/rtc_base/third_party/sigslot/README.chromium
+++ b/rtc_base/third_party/sigslot/README.chromium
@@ -3,7 +3,7 @@ Short Name: sigslot
 URL: http://sigslot.sourceforge.net/
 Version: 0
 Date: 2018-07-09
-License: Custom license
+License: Ignorable
 License File: LICENSE
 Security Critical: yes
 Shipped: yes
